### PR TITLE
ZUR-130 My PR adds resizes footer social icons

### PR DESCRIPTION
**Pull request by bimbo-ade**
This PR resizes the footer social icon

before
before-footer-edit

after

after-footer-edit2

### CODE REVIEW

**Comment by bimbo-ade** [comment]
@JulianaSau hi can you please check this out

**Comment by JulianaSau** [comment]
Why did you delete the pr template?

**Comment by bimbo-ade** [comment]
> Why did you delete the pr template?

there was so much error, it was giving a blank page on my browser, 
so i had to reclone the repo and start all over again.
i hope i did not do anything wrong with that decision?

**Comment by JulianaSau** [comment]
That's not what I mean. Check how this pr is structured. https://github.com/zurichat/zc_main/pull/2624#issuecomment-1313190164

**Comment by bimbo-ade** [comment]
oh yh, i saw that, i did not think it through before deleting. I apologize, is there a way to go back on my actions

**Comment by JulianaSau** [comment]
And update your branch

**Comment by JulianaSau** [comment]
It's alright. I'll just check your pr

**Comment by bimbo-ade** [comment]
Thank you, and i sincerely apologize

File: yarn.lock [hunk 1]
```diff
@@ -14650,7 +14650,11 @@ react-focus-lock@^2.5.2:
...
 react-giphy-searchbox@^1.5.4:
   version "1.5.4"
+<<<<<<< HEAD
+  resolved "https://registry.npmjs.org/react-giphy-searchbox/-/react-giphy-searchbox-1.5.4.tgz"
+=======
   resolved "https://registry.yarnpkg.com/react-giphy-searchbox/-/react-giphy-searchbox-1.5.4.tgz#49c6f98bf1afefbbd3a9ab3c5f79709cb32f399b"
+>>>>>>> 2154bd17a33cba724410a7498a03cdbc228643b3
   integrity sha512-5SCiT3Y/0aeJcFfd3yXwJ1LShmDTTSL/aKJUXu9t0eBvwUU5DSev/JvMx3ijedsWpHuMtkwHLy5jV3pBDdzPQw==
   dependencies:
     bricks.js "^1.8.0"
```
Comment (JulianaSau) [inline yarn.lock:14660]: Resolve these conflicts please
remove this 
```
<<<<<<< HEAD
  resolved "https://registry.npmjs.org/react-giphy-searchbox/-/react-giphy-searchbox-1.5.4.tgz"
=======
```
and this `>>>>>>> 2154bd17a33cba724410a7498a03cdbc228643b3`

You should not be editing the `yarn.lock` files

**Review by JulianaSau — CHANGES_REQUESTED** [review]

**Comment by bimbo-ade** [comment]
hi, i have implemented the changes requested

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**